File: isPrime.c
#include <stdio.h>
typedef enum { false, true } bool;
int primes =0;

bool isPrime(int number){
   int i;
   for (i = 2; i < (number / 2 ); i++ ){
       if (number % i == 0){
           return false;
       }
   }
   return true;
}

int main(){
    int k;
    for (k = 3; k < 2000000; k++){
        if (isPrime(k) == true){
           primes++;
        }
     }
     printf("\n%d\n", primes);
}

File: isPrime.pl
$primes = 1;
sub isPrime{
   my $number = @_[0];
   for ($i = 2; $i < $number; $i++ ){
       if ($number % $i eq 0){
           return 0;
       }
   }
   return 1;
}
for ($k = 3; $k < 2000000; $k++){
    if (isPrime($k) eq 1){
       $primes++;
    }
}
print " $primes \n";
